keyboards/basekeys/slice/rev1_rgb/keymaps/via/rules.mk

```
@@ -0,0 +1,4 @@
+TAP_DANCE_ENABLE = no
+VIA_ENABLE = yes
+BACKLIGHT_ENABLE = no
+SPACE_CADET_ENABLE = no
```

```suggestion
VIA_ENABLE = yes
```
I don't see these enabled in the keyboard-level `rules.mk`.

---

I committed these changes.